rest/tests/test_learning_resource.py

```diff
@@ -451,3 +547,123 @@ def test_static_assets_delete(self):
             self.DEFAULT_LR_DICT
         )
         self.assertEqual(resp.status_code, HTTP_405_METHOD_NOT_ALLOWED)
+
+    def test_learning_resource_exports(self):
+        """Test for creating learning resource ids in session shopping cart"""
+
+        # setup our data
+        self.logout()
+        self.login(self.add_repo_user)
+
+        repo_slug1 = self.repo.slug
+        repo_slug2 = self.create_repository()['slug']
+        repo2 = Repository.objects.get(slug=repo_slug2)
+
+        # 'add_repo_user' will be able to see both repos
+        assign_user_to_repo_group(
+            self.add_repo_user, self.repo, GroupTypes.REPO_ADMINISTRATOR)
+        self.import_course_tarball(repo2)
+
+        repo1_lrid1 = get_resources(self.repo.id).all()[0].id
```

Comment:
This is a nit. These two lines can use `.first()` instead of `.all()[0]`. Given the `.all()[1]` below an argument could be made for uniformity. Feel free to ignore this if you think it's more readable this way.